package.json

```diff
@@ -13,7 +13,7 @@
     "docker:dev": "npm run dev",
     "format": "prettier --write \"**/*.{cjs,js,json,md}\"",
     "format:check": "prettier --check \"**/*.{cjs,js,json,md}\"",
-    "git:pre-commit-hook": "npm run format:check && npm run lint",
+    "git:pre-commit-hook": "npm run format:check && npm run lint && npm test",
```

Comment:
do we not have husky for this?

Reply:
Yeah this is being used by husky. It was abstracted it out into an `npm` script that is called by husky - https://github.com/DEFRA/cdp-node-backend-template/blob/main/.husky/pre-commit#L1